pac-update.sh

```diff
@@ -0,0 +1,131 @@
+set -e
+      
+echo "        ..........    												 "                                                  
+echo "      .............. 												     "                                                
+echo "   ....................   											 "                                            
+echo "  .....-yyyyyyys+-:oo-..       oooooooo.      ooooo         oooooooo   " 
+echo " ......:MMMMNNdddmMMMd...     MMMMNMMMMMh.   MMMMMMM      MMMMMMMMMM   "
+echo "......:MMmy..-+MNdh:.....     MMMM: .dMMMy  oMMMyMMNN   .NMMMMy        "
+echo "......:ddmm...-hdNm......     MMMM+-+mMMMs :MMMs dMMm   yMMMMo         "
+echo "....:ohNMMNdoydNMMN/.....     MMMMMMMMMmo. NMMN  -MMMd  yMMMM.         "
+echo "...+dMNdddNmmdhy+-.......     MMMM+::-    dMMMNmmmMMMMs :MMMMm.        "
+echo "......::MMMm.............     MMMM:      yMMMdoooooNMMM  mMMMMNdddN+   "
+echo " ......:MNdo.............     dddd-     ddddd.     ddddd.  :ohdmmdhy:  "
+echo "  ....-+-..............                                                "
+echo "    .................                                                  "
+echo "       ...........  													 "
+
+echo "################################################"
+echo "#         Welcome PAC wallet update		     #"		
+echo "################################################"
+echo "" 
+echo "This script will update PAC to the latest version (xxx)"
+echo "Before running the script we recommend to backup your private keys and your wallet.dat file that is under ./paccoincore/wallet.dat even though this script will not affect them"
+
+read -p "Do you want to autobackup wallet.dat into the current path and continue with the process? [y/n]:" cont
+if [ $cont = 'y' ] || [ $cont = 'yes' ] || [ $cont = 'Y' ] || [ $cont = 'Yes' ]; then
+	currpath=$( pwd )
+	echo "Backing up the wallet.dat into $currpath"
+	cp .paccoincore/wallet.dat "$currpath/pacwalletbackup.dat"
+	echo "Wallet.dat backup is on: $currpath"
+else
+	exit
+fi
+
+
+if [ -e ~/paccoin-cli ] && [ -e ~/paccoind ]; then
+	echo ""
+	echo "#######################################"
+	echo "#   Stopping pac services (daemon)    #"		
+	echo "#######################################"
+	echo ""
+
+	wallstatus=$( ./paccoin-cli mnsync status ) || true
+	if [ -z "$wallstatus" ]; then
+		echo "Daemon is not running no need to stop it"
+	else
+		echo "Stopping the Daemon"
+		./paccoin-cli stop
+		sleep 60
+		
+		wallstatus=$( ./paccoin-cli mnsync status ) || true
+		if [ -z "$wallstatus" ]; then
+			echo "Daemon stopped!"
+		else
+			echo "Daemon can not be stopped! Please kill the deamon process or run ./pacoin-cli stop, then run the script again"
+			exit
+		fi
+	fi
+
+	echo ""
+	echo "###############################"
+	echo "#   Removing old binaries    #"		
+	echo "###############################"
+	echo ""
+
+	echo "Removing paccoin-cli"
+	rm paccoin-cli
+
+	echo "Removing paccoind"
+	rm paccoind
```

Comment:
instead of removing it I'd ask the user if he/she wants to back it up, this in case of any inconvenience it'd be easy to restore the old daemon, what do you think?

Reply:
I think we should leave it that way, the next this to do if something happens when downloading/running the new daemon, is to try running again this script.